src/main/java/com/google/sps/servlets/CheckStudentStatus.java

```diff
@@ -53,10 +53,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
       // Find position in queue
       ArrayList<String> queue = (ArrayList) classEntity.getProperty("studentQueue");
-      String pos = Integer.toString(queue.indexOf(uID) + 1);
+      String studentPosition = Integer.toString(queue.indexOf(studentID) + 1);
```

Comment:
You shouldn't need to convert to a string `PrintWriter.print` accepts an `int`

https://docs.oracle.com/javase/7/docs/api/java/io/PrintWriter.html#print(int)